libs/checkpoint/src/serde/jsonplus.ts

```diff
@@ -140,6 +140,12 @@ function _default(_key: string, obj: any): any {
     return _encodeConstructorArgs(RegExp, undefined, [obj.source, obj.flags]);
   } else if (obj instanceof Error) {
     return _encodeConstructorArgs(obj.constructor, undefined, [obj.message]);
+    // TODO: Remove special case
+  } else if (obj?.lg_name === "Send") {
```

Comment:
how was this not handled already?

Reply:
I am actually not sure, I suppose the default JSON.stringify did something good. It serialized with `lg_name` though so this just removes that to preserve immutability.